build.sbt

```diff
@@ -44,7 +44,9 @@ libraryDependencies += "org.yaml" % "snakeyaml" % "1.17"
 
 libraryDependencies += "com.typesafe.scala-logging" %% "scala-logging-slf4j" % "2.1.2"
 
-resolvers += "Apache Development Snapshot Repository" at "https://repository.apache.org/content/repositories/snapshots"
+// Use RC5 until public release is available
```

Comment:
Should we just wait a couple more days for 2.4 to be out?

Reply:
I just updated it to use the official release.